hat/api_import/models.py

```diff
@@ -11,6 +17,19 @@
 )
 
 
+def api_import_upload_to(api_import: "APIImport", filename: str):
+    today = date.today()
```

Comment:
```suggestion
    today = timezone.now().date()
```

Nitpick but this is an old habit of mine: always using UTC (`from django.utils import timezone`) inside the codebase.

`timezone.now().date()` ensures we align with our configured timezone rather than the server's physical location.